tools/run_tests/dockerize/build_interop_image.sh

```diff
@@ -75,6 +75,22 @@ git branch -f jenkins-docker
 
 CONTAINER_NAME="build_${BASE_NAME}_$(uuidgen)"
 
+# build_interop.sh could be missing from old git tree and we will have issue unless we find the proper
+# build_interop.sh to use in such case.
+if ! [ -e $GRPC_ROOT/tools/dockerfile/interoptest/$BASE_NAME/build_interop.sh ]; then
```

Comment:
We need to build an older client and we have a) code to compile; b) Dockerfile; 3) build_interop.sh.  

Given that we introduced a runtime variation, b and c don't exist on the old tree.  We have two options:
A) Use b and c from head for lang_runtime;
B) Use b from head and c from old tree.

I first tried Option A, but ran into problem because build_interop.sh has changed over time and no longer works as-is against old tree.  We could add logic to fix build_interop.sh or find an alternative from old tree which is Option B.

In Option B, we don't have the lang_runtime/build_image.sh, so the closest we can find is lang/build_image.sh.  This is not ideal, but it guaranteed to work.

If we go with Option A, we need to either fix build_image.sh from head by bringing version specific build_image.sh logic to head.

What do you prefer among all these no optimal solutions?